kodilla-patterns/src/main/java/com/kodilla/patterns/strategy/social/User.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.patterns.strategy.social;
+
+public class User {
+    final private String name;
+    protected SocialPublisher socialPublisher;
```

Comment:
czemu jedno pole jest final i private a drugie po prostu protected ??

Reply:
Tak było też w materiale teoretycznym ;) Rozumiem, że chodzi tu o widoczność tego pola w pakiecie - klasy dziedziczące po klasie "User" korzystają z niego. Ale z pola "name" również, a ono nie jest "protected", tylko "private", więc już nie wiem o co chodzi :/

Reply:
jeśli tak jest w materiałach to OK - ja nie do końca popieram tutaj użycie tego modyfikatora. Zostawmy to już tak (omówione podczas video)